src/ui/components/TestSuite/views/TestRecording/TestRecordingEvents/UserActionEventRow.tsx

```diff
@@ -150,15 +151,19 @@ export default memo(function UserActionEventRow({
 
   return (
     <div
-      className={styles.Row}
+      className="flex items-start space-x-0"
```

Comment:
If you need help with the styling, I'm happy to help, but let's avoid mixing Tailwind and CSS modules in a view that's CSS modules.

Reply:
From the description of this PR:

Code note:

I'm using tailwind. I tried doing this in a css module but something was overwriting it and I couldn't figure it out for the life of me. So I'm submitting this as-is, and if we feel strongly about the tailwind stuff, any help would be welcome.

Reply:
Yup, I saw the code note, and I do feel strongly enough about it to leave the comments. If the work around I suggested isn't sufficient then talk to me and let's figure out what's needed to avoid mixing Tailwind and CSS modules. (I don't want that to land.)

Reply:
No worries, hopefully your code works and if not I'll tell you